src/components/application_manager/rpc_plugins/sdl_rpc_plugin/src/commands/mobile/delete_interaction_choice_set_request.cc

```diff
@@ -53,9 +55,14 @@ DeleteInteractionChoiceSetRequest::DeleteInteractionChoiceSetRequest(
                          application_manager,
                          rpc_service,
                          hmi_capabilities,
-                         policy_handler) {}
+                         policy_handler)
+    , response_result_code_(
+          hmi_apis::Common_Result::eType::UNSUPPORTED_RESOURCE)
+    , response_result_(false) {}
 
-DeleteInteractionChoiceSetRequest::~DeleteInteractionChoiceSetRequest() {}
+DeleteInteractionChoiceSetRequest::~DeleteInteractionChoiceSetRequest() {
+  unsubscribe_from_all_events();
```

Comment:
@IGapchuk please check dtor in base class. As I remember this function is called somewhere there

Reply:
@AKalinich-Luxoft Deleted in e32b004ae4d7e84e153d6b8082cb3c1a800a41ec.